bundles/Expense/Resources/assets/new.js

```diff
@@ -13,7 +13,9 @@ $(() => {
 
     const $projectCtrl = $('#form_project');
 
-    const buildIssuePicker = () => {
+    const renderIssue = issue => issue.summary + ' (' + issue.key + ')'
```

Comment:
Semicolons in js files...

Reply:
Sorry, I'll remove the superfluous semicolon in line 14 …